test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -288,7 +323,14 @@ int run_test_with_two_queue(cl_context &context, cl_command_queue &cmd_queue1,
 
                 if (iter != (maxIter - 1))
                 {
-                    clCl2VkExternalSemaphore->signal(cmd_queue2);
+                    if (use_fence)
+                    {
+                        fence->wait();
```

Comment:
Need clFinish here instead of fence wait to ensure OpenCL work is done before starting vulkan work  submission.